src/main/java/utility/UtilJobs.java

```diff
@@ -37,7 +37,7 @@ public static Job createJobObjFromJson(JSONObject json) {
                 .url((String) json.get("url"))
                 .classs((String) json.get("class"))
                 .filesUrl((String) json.get("files-url"))
-                .retCode((String) json.get("retCode"))
```

Comment:
can you keep the camalCase for the variable name? keeps it consistent variable name pattern like the others..

so .retCode((String) json.get("retcode"))

should still work.